packages/salesforcedx-vscode-core/src/commands/baseDeployRetrieve.ts

```diff
@@ -78,7 +84,15 @@ export abstract class DeployRetrieveExecutor<
         status === RequestStatus.SucceededPartial
       );
     } catch (e) {
-      throw formatException(e);
+      if (e.name === 'SourceConflictError') {
+        this.handleSourceConflictError(e);
+        return true;
+      } else {
+        // Error, but not a Source Conflict Error.  Prior to adding
+        // SourceTracking, this was the only statement in the catch
+        // block.
+        throw formatException(e);
+      }
     } finally {
       await this.postOperation(result);
     }
```

Comment:
Since this method returns a boolean, at line 99 it should probably return false.

Reply:
As per our convo in 11sies mtg today, no further changes needed here.  Resolving conversation.